src/main/features/dashboard/views/macro/claimStatus/claimantResponse.njk

```diff
@@ -0,0 +1,33 @@
+{% macro claimantRejectedDefendantAsCompanyOrOrganisationResponse() %}
+<p>
+  {{ t('You need to send the defendant’s financial details to the court.') }}
+</p>
+{% endmacro %}
+
+{% macro claimantRejectedDefendantAsCompanyOrOrganisationResponseDetailedStatus(claim) %}
+
+{% set claimNumber = claim.claimNumber %}
+
+<h2 class="heading-small">{{ t('You need to send the defendant’s financial details to the court') }}</h2>
+
+<p>{{ t('Make sure you include your claim number: {{ claimNumber }}.', { claimNumber: claimNumber }) }}</p>
+<p>{{ t('The court will review the details to see if the defendant can afford your repayment plan.') }}</p>
+
+<h3 class="heading-small">{{ t('Use this address') }}</h3>
+<p>{{ t('The defendant should have sent you the details - send them to:') }}</p>
+<p>
+  Civil Money Claims<br/>
+  St. Katharine's House<br/>
+  21-27 St. Katharine's Street<br/>
+  Northampton<br/>
+  NN1 2LH
+</p>
+
+<p>{{ t('Or email them to moneyclaims@hmcts.gsi.gov.uk') }}</p>
```

Comment:
this should be the configurable e-mail address in case we change it.

Reply:
Applied support email address like other places